http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/JsonPayloadIT.java

```diff
@@ -78,67 +81,64 @@ public void sendLargeComplexJSONPayload() throws IOException {
                 .get("/persons")
                 .then()
                 .statusCode(OK_STATUS_CODE)
-                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages."));
+                .body(containsString("Hello, Singleton Middleton! You have 4 unread messages.")).extract().response();
```

Comment:
I don't see point why we extract response if it isn't save to variable.

Reply:
good catch, fixing it, thanks.